src/pocketmine/command/CommandReader.php

```diff
@@ -91,6 +95,10 @@ public function run(){
 
 			$lastLine = microtime(true);
 		}
+		global $stdin;
+		if(is_resource($stdin)){
+			fclose($stdin);
+		}
```

Comment:
robske110, I was indicating that I had seen and was following with interest your conversation with dktapps.